### DISCUSSION TITLE
Screen sharing moderator actions

### DISCUSSION THREAD

**Discussion by Astagor** [edumeet/edumeet#849] [Ideas]
I see that now the moderator can mute for all video and audio for a chosen peer. Now if the peer except of the camera had screen sharing, the mute video for all turns off the camera and screen sharing. This is also fine. However, when the user enables again only the screen sharing, the mute video for all button does not appear and the moderator cannot mute it for a second time.
Solutions:
- make a separate moderator control button for screen sharing
or
- make the 'mute video for all' appear when either there is a camera or a screen sharing feed.

**Comment by L4ky** [comment]
The "mute video for all" is always there for me.

I noticed that this issue happens for the "stop video for peer" icon.
I'll work on that.

**Comment by Astagor** [comment]
No, you got me wrong. I not writing about the large button on top, I'm writing about the small 'per peer' buttons.

**Comment by L4ky** [comment]
Ok, i misread probably.
I'll work on it.

**Comment by L4ky** [comment]
Fixed in https://github.com/havfo/multiparty-meeting/pull/355

**Comment by misi** [comment]
We need to add moderator action "disable all screensharing" visible only if there is at least one screensharing in the room otherwise hidden.
